source/sync/configure/enable-sync.txt

```diff
@@ -179,33 +179,27 @@ Procedure
                  -X GET \
                  -h 'Authorization: Bearer <Valid Access Token>'
 
-            Once you have the configuration, add the ``flexible_sync`` object with the
-            following template configuration:
+            For details on the sync configuration, refer to the :ref:`sync-configuration-reference`.
 
-            .. code-block:: json
+         .. step:: Enable Development Mode (optional)
 
-               {
-                 ...
-                 "flexible_sync": {
-                   "state": "enabled",
-                   "database_name": "<Name of Database>",
-                   "client_max_offline_days": <Number>,
-                   "is_recovery_mode_disabled": <Boolean>,
-                   "indexed_queryable_fields_names": <Array of String Field Names>,
-                   "queryable_fields_names": <Array of String Field Names>,
-                   "collection_queryable_fields_names": <Map[String][]String>
-                 }
-                 ...
-               }
+            You may need to enable :ref:`Development Mode <development-mode>` before deploying your changes. 
+            You can use the following command to enable Development Mode:
```

Comment:
I assume this is the trouble-causing endpoint we're not actually exposing/documenting?

Reply:
yes :(